examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,59 @@
+"""
+=============================
+Plot positions on a blank map
+=============================
+
+This example showcases how to plot positions on a blank map.
+It is commonly seen in papers and presentations to show HPC positions of events that occurred at
+different times and therefore no single observation is appropriate.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import frames
+
+################################################################################
+# First we will create a blank map using with an array of zeros.
+# Since there is no WCS information, we will need to construct a header to pass to Map.
+data = np.full((1000, 1000), 0)
```

Comment:
Not a big deal, but why use `np.full()` instead of `np.zeros()`?

Reply:
Okay, will change that